### ISSUE TITLE
Unable to delete update server double

### ISSUE THREAD

**Issue by cmirnow** [joomla/joomla-cms#44204]
In the dashboard of my Joomla site (5.1.4), after numerous updates (it's an old site) there are two identical update servers. I'm trying to delete one of them, but it's not working. Can you tell me what actions are available to me to remove this extra server? Is this a bug or a feature?

![Image](https://github.com/user-attachments/assets/4ab77392-27ba-4743-b299-cb8998825084)

![Image](https://github.com/user-attachments/assets/0acdf2ca-19e1-45cc-a305-765d14be03f7)

**Comment by fgsw** [comment]
What i would do:
1. Do a backup
2. Open in my database the table ```*_update_sites``` in phpMyAdmin
3. Delete the row

Having more questions ask in [Forums](https://forum.joomla.org) and close this issue.

**Comment by richard67** [comment]
> What i would do:
> 
> 1. Do a backup
> 2. Open in my database the table `*_update_sites` in phpMyAdmin
> 3. Delete the row

There is also a table update_sites_extensions which holds the cross references fro update_sites to extensions. That should also be checked.

It is by design that you cannot delete a core update site, and the rebuild button also doesn’t touch core sites.

But maybe that should be changed/improved. I know of a few other cases where a site had a long update history and duplicate update sites for the core.

**Comment by brianteeman** [comment]
> But maybe that should be changed/improved. I know of a few other cases where a site had a long update history and duplicate update sites for the core.

For non protected update sites such as joomla core a rebuild of the update sites should resolve any duplicates

**Comment by brianteeman** [comment]
Rebuild should delete all the update sites and then rebuild the table from the extension xml

**Comment by richard67** [comment]
> Rebuild should delete all the update sites and then rebuild the table from the extension xml

@brianteeman I agree that it should do that, but it doesn't. Any update site which refers to the core (joomla files extension) will not be touched. I had once investigated a similar problem and tried it. Reading the code, especially the comment, made me then believe it is desired behaviour, so I haven't opened an issue.

**Comment by cmirnow** [comment]
> I agree that it should do that, but it doesn't.

@brianteeman , I opened this issue because the “rebuild” button does not solve the problem of removing the update server double. My opinion today is that this is a bug that needs to be fixed. But maybe the development community will decide otherwise.
